src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/SunPKCS11.java

```diff
@@ -87,6 +102,11 @@ public final class SunPKCS11 extends AuthProvider {
 
     static NativeResourceCleaner cleaner;
 
+    // This is the SunPKCS11 provider instance
+    // there can only be a single PKCS11 provider in
+    // FIPS mode.
+    static SunPKCS11 mysunpkcs11 = null;
```

Comment:
Please remove this redundant initialization.

Reply:
done